docs/design/domain-class-types-in-kotlin-sdk.md

```diff
@@ -18,7 +18,7 @@ implementations for model shapes.
 
 The Smithy [weather example's](https://awslabs.github.io/smithy/quickstart.html#complete-example) `GetCity` operation is
 used as an example model description. The operation takes a `GetCityInput` and returns a `GetCityOutput`. We will focus
-on the input type below to show how each approach is expressed and responds to change.
+on the input type below to show how normal and data classes would be generated and respond to model updates.
```

Comment:
Agreed that it's awkward but think it's better as is because separating them is the point of the comparison of approaches in the doc.  I want to come up w/ a better term than `normal`...ideas?

Reply:
Taxonomically, there's nothing special about "normal" classes except they're not data classes. So maybe "non-data classes"? Grammatically, that's still awkward so it's probably better to say "...to show how both (options|approaches|proposals) would be generated and respond...".

Reply:
regular vs data? I mean I agree with Ian there isn't really a difference but since it's the point of the doc to discuss using data classes vs not we need a term. Normal, regular, etc all seem fine to me FWIW.

Reply:
I am using the terms in the section headers as well.  For clarity what is mentioned here should be consistent w/ the headers.  I'm updating the sentence to the following for that:

```
...on the input type below to show how both classes and data classes would be generated and respond to model updates.
```